docs/source/serving/deploying_with_lws.rst

```diff
@@ -0,0 +1,11 @@
+.. _deploying_with_lws:
+
+Deploying with LWS
+============================
+
+LeaderWorkerSet (LWS) is low level API aims to address common deploy patterns of AI/ML inference workloads,
+especially designed for multi-host distributed inference needs.
+
+vLLM can be deployed with `LWS <https://github.com/kubernetes-sigs/lws>`_ on Kubernetes for distributed model serving.
+
+Please see `this guide <https://github.com/kubernetes-sigs/lws/tree/main/docs/examples/vllm>`_ for more details on using vLLM with LWS.
```

Comment:
this link is a 404

Reply:
The correct link is https://github.com/kubernetes-sigs/lws/tree/main/docs/example/vllm

Reply:
It will take effect after https://github.com/kubernetes-sigs/lws/pull/145, sorry for the confusion.